CNAME

```diff
@@ -1 +0,0 @@
-geoserver.org
```

Comment:
isn't the CNAME file required to make github hosting work? I forget...

Reply:
It is now generated via a config option in the workflow script.